lib/Ohmbrewer_RIMS.h

```diff
@@ -105,6 +85,24 @@ namespace Ohmbrewer {
             // friend std::ostream& operator<<( std::ostream& os, RIMS const& rims);
 
             /**
+             * The Tube thermostat
+             * @returns The Thermostat object representing the RIMS tube elements
+             */
+            Thermostat* getTube() const;
+
+            /**
+             * The temperature sensor located in the tun
+             * @returns The Temperature Sensor object representing the sensor located in the mash tun
+             */
+            TemperatureSensor* getTunSensor() const;
```

Comment:
not in RIMS, in rims it stays as tun sensor, and actually safety sensor is the other one anyways